app/src/main/java/com/digitalarchitects/rmc_app/network/RmcApiService.kt

```diff
@@ -0,0 +1,26 @@
+package com.digitalarchitects.rmc_app.network
+
+import com.digitalarchitects.rmc_app.model.User
+import com.digitalarchitects.rmc_app.model.Vehicle
+import retrofit2.http.GET
+
+
+interface RmcApiService {
+    /**
+     * Returns a [List] of [User] and this method can be called from a Coroutine.
+     * The @GET annotation indicates that the "photos" endpoint will be requested with the GET
```

Comment:
"photos"? Do you mean Users

Reply:
Yes I meant users, comment is copied and pasted from one of the Android Developers codelab. XD